katdal/dataset.py

```diff
@@ -358,10 +358,11 @@ class DataSet:
 
     """
 
-    def __init__(self, name, ref_ant='', time_offset=0.0):
+    def __init__(self, name, ref_ant='', time_offset=0.0, url=''):
```

Comment:
The base `DataSet` has tons of attributes that are not passed as parameters to the initialiser. I'm hesitant to add `url` at this level because it's kind of arbitrary. In fact, I'm thinking of getting rid of the rest in a future release too.